browser-extension/mv2/src/background/background.js

```diff
@@ -1073,9 +1073,15 @@ BG.Methods.onContentScriptLoadedNotification = async (tabId) => {
   }
 
   if (window.tabService.getData(tabId, "recordSession") === true) {
-    chrome.tabs.sendMessage(tabId, { action: RQ.CLIENT_MESSAGES.START_RECORDING }, () =>
-      window.tabService.removeData(tabId, "recordSession")
-    );
+    chrome.tabs.sendMessage(tabId, { action: RQ.CLIENT_MESSAGES.START_RECORDING }, () => {
+      window.tabService.removeData(tabId, "recordSession");
+      BG.Methods.showToast(
```

Comment:
Since this is a client-side logic and we already sending `START_RECORDING` event, can we simply pass an event param `notify: true` and let content script render and show this message?